lib/src/credential_service.dart

```diff
@@ -49,29 +61,25 @@ class CredentialService {
         discoveryDocs: [_dv3DiscoveryUrl, _dbmDiscoveryUrl],
         scope: _scope);
 
-    await GoogleAPI.client.init(initArgs).then(allowInterop((value) {
+    final completer = Completer<bool>();
```

Comment:
just a thought, is it possible to move the completer logic into your wrapper class? would be nice to abstract that detail away from the caller.

Reply:
Good point!